### PR TITLE
[PC-854] Benjamindannegard/max carrier x8 gs

### PR DESCRIPTION
**Pull request by BenjaminDannegard**
## What This PR Changes
- Added tutorial for max carrier and x8 getting started
## Contribution Guidelines
- [ ] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Comment by marqdevx** [comment]
Preview not working, gonna try locally

File: content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/x8-getting-started/content.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+One of the many features of the Portenta Max Carrier is the Murata CMWX1ZZABZ-078 that enables LoRaWAN® connectivity. LoRaWAN® is a Low Power Wide Area Network (LPWAN) designed to connect low power devices to the Internet. It was developed to meet and fulfill Internet of Things (IoT) devices' requirements, such as low-power consumption and low data throughput.
+
+The appropriate antenna and frequencies depends on the region. The common frequencies are 915 MHz for North America and Australia, and 863 MHz for European region. Frequencies are on a range, so for example Australia region it is possible to use 928 MHz compatible antenna and configuration.
+
+For more in-depth information about LoRa® and LoRaWAN®, please read [The Arduino Guide to LoRa® and LoRaWAN®](https://docs.arduino.cc/learn/communication/lorawan-101).
+
+### Ethernet
+
+The Gigabit Ethernet physical interface is directly connected to the high density connector to the Portenta X8 board. The connector includes a LED for indicating activity using the color orange , there is also a LED using the color green to indicate speed.
```
Comment (marqdevx) [inline content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/x8-getting-started/content.md:100]:
```suggestion
The Gigabit Ethernet physical interface is directly connected to the high density connector to the Portenta X8 board. The connector includes a LED for indicating activity using the color orange, there is also a LED using the color green to indicate speed.
```

Comment (marqdevx) [inline content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/x8-getting-started/content.md:100]:
```suggestion
The Gigabit Ethernet physical interface is directly connected to the high density connector to the Portenta X8 board. The connector includes a LED for indicating activity using the color orange, there is also a LED using the color green to indicate speed.
```

**Review by marqdevx — APPROVED** [review]
LGTM

**Comment by marqdevx** [comment]
Question: Do we need to show this tutorial o the X8 page too? 🤔

File: content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/x8-getting-started/content.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+- Learn about the Portenta Max Carrier's headers, peripherals and connectors
+
+### Required Hardware and Software
+
+- [Arduino® Portenta X8](https://store.arduino.cc/products/portenta-x8)
+- [Arduino® Portenta Max Carrier](https://store.arduino.cc/products/portenta-max-carrier)
+
+## The Portenta Max Carrier
+
+The Portenta Max Carrier was designed to add capabilities to the Arduino® Portenta X8 board and provide easy access to its on-board peripherals. It is designed to enable research and develop industrial grade advanced applications, from fast prototyping to deployable robust single board computer.
```
Comment (jhansson-ard) [inline content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/x8-getting-started/content.md:36]: We can remove the "Arduino" after mentioning it the first time (add this in the introduction and remove from the rest) ⭐

File: content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/x8-getting-started/content.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+|                        | 31                                                                 | mPCIe_TX_N  | Diff.   | PCIe Transmission Differential Negative |
+|                        | 32                                                                 | SDA         | Digital | I2C Data  Line                          |
+|                        | 33                                                                 | mPCIe_TX_P  | Diff.   | PCIe Transmission Differential Positive |
+|                        | 36                                                                 | USB_DN      | Diff.   | USB Data Differential Negative          |
+|                        | 38                                                                 | USB_DN      | Diff.   | USB Differential Positive               |
+
+
+## Conclusion
+
+Now you should know more about the possibilities of the Portenta Max Carrier and the Portenta X8.
```
[the remark this answers is not in the document]
Comment (BenjaminDannegard) [reply content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/x8-getting-started/content.md:216]: The tutorial is kind of a showcase of the different peripherals so no real singular practical application is being shown in the tutorial. I can see if I can flesh this part out a bit more.

[the remark this answers is not in the document]
Comment (jhansson-ard) [reply content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/x8-getting-started/content.md:216]: Yes, either that or maybe removing the section if it doesn't make sense?

**Pull request merged**